boards/xtensa/heltec_wifi_lora32_v2/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_PINMUX_ESP32)
+if(CONFIG_PINCTRL_ESP32)
```

Comment:
seems like board is using just GPIO API to configure some pins, how is that related to pinctrl?

Reply:
Good point. I have changed it to evaluate `CONFIG_GPIO_ESP32` instead.